changelog/log_float_double_implementations.dd

```diff
@@ -0,0 +1,15 @@
+Single- and double-precision implementations for log function families
+
+New `float` and `double` overloads of $(REF log, std, math, exponential),
+$(REF log10, std, math, exponential), $(REF log1p, std, math, exponential),
+ $(REF log2, std, math, exponential), and $(REF logb, std, math, exponential)
+have been added to Phobos with proper 'software' implementations in the
+corresponding precision. Furthermore, $(REF logb, std, math, exponential) is
+now `pure`.
+
+While this may result in a slowdown in some cases for DMD, the overall speed-up
+factor for GDC and LDC is over 3x, for both `double` and `float`.
```

Comment:
Heh I guess that's a leftover from the other PR, where I performed some bench runs to come up with actual numbers.

Reply:
I assume it'll be the same because x87 vs xmm speeds are usually universal across the board (IIRC it's the constant push/popping off the x87 stack that is the slowest. ;-)

Also, I imagine the cycle count for fyl2x (used by DMD) would be a lot less than our software implementation, hence slowdown for DMD.

Side note, does it make sense to even keep the inline yl2x implementations?  The result you get from it is not even IEEE compliant.